test/configs/zombieStarlight.json

```diff
@@ -17,8 +17,7 @@
                                 "max_candidate_depth": 3
                             },
                             "scheduler_params": {
-                                "scheduling_lookahead": 2,
-                                "num_cores": 1
+                                "scheduling_lookahead": 2
```

Comment:
Here, for instance

Reply:
What do you mean?

Reply:
instead of hardcoding the number of blocks to 4 into genesis, I would rather have it as it was and then control it in zombienet through here, what do you think?

Reply:
I can add it here as well, but I would also leave the value of 4 in genesis.

Reply:
fine, but yes let's add it in both places

Reply:
fixed